psi4/src/psi4/dfocc/tei_grad.cc

```diff
@@ -199,15 +198,15 @@ void DFOCC::tei_grad(std::string aux_type) {
             int oN = primary_->shell(N).function_index();
 
             int ncart = cP * cM * cN;
```

Comment:
`ncart` needed?

Reply:
occ and dfocc we live alone on small matters until we can reconcile with Ugur.